homeassistant/components/modbus/climate.py

```diff
@@ -70,16 +77,17 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     modbus_slave = config.get(CONF_SLAVE)
     target_temp_register = config.get(CONF_TARGET_TEMP)
     current_temp_register = config.get(CONF_CURRENT_TEMP)
```

Comment:
Please use `dict[key]` here as well since these are required config keys.

Reply:
OK, I will update the code.

One question though: Is this a preferred way to do it onwards? I'm asking because I went through several integrations to see the code, and they all use `config.get()`. If I understand correctly, the purpose is to make the code slightly more effective because we have a guarantee that the value is set thanks to validation. Is it right?

Reply:
Thank you for cleaning this up 👍 

Yes, indeed that is the preferred way of doing it onwards. `config.get()` should only be used with optional config keys that do not have a default value. 

> If I understand correctly, the purpose is to make the code slightly more effective because we have a guarantee that the value is set thanks to validation. Is it right?

Absolutely correct.

Reply:
OK, now it should be ready :) I will probably create another PR just to clean up other Modbus entities in a similar way.